### DISCUSSION TITLE
Is it possible to write an connection Authentication middleware?

### DISCUSSION THREAD

**Discussion by JasterV** [Totodore/socketioxide#269] [Q&A] [answered]
I want to implement the following: https://socket.io/docs/v3/middlewares/#handling-middleware-error

Basically if the client provides wrong Auth data, I would like a middleware to handle that and return to the client a `connect_error` like its explained on the official documentation of Socket.io

How would I do that using `socketioxide`? I am using Axum :)

**Comment by Totodore** [comment, marked as answer]
Currently it is not possible. I will add a tracking issue for this functionality.

**Reply by Totodore** [reply]
Issue #270

**Reply by JasterV** [reply]
Thank you!
